src/deviceinfo_msm8916

```diff
@@ -0,0 +1,6 @@
+deviceinfo_flash_offset_base="0x80000000"
```

Comment:
These offsets aren't used?

Reply:
ah i just copied them from pmos. i can remove them

Reply:
ah sorry i misunderstood you. they are in use:
https://github.com/dreemurrs-embedded/Jumpdrive/blob/master/Makefile#L84

Reply:
the code already was in place for sdm845